js/src/View/TodoView.js

```diff
@@ -0,0 +1,53 @@
+var Backbone = require('backbone');
+var $ = require('jquery');
+var _ = require('underscore');
+var handlebars = require('handlebars');
+
+var TodoCollection = require('../Collection/TodoCollection.js');
+var TodoModel = require('../Model/TodoModel.js');
+
+var todoViewTemplate = require('../templates/todo.hbs');
+
+
+var TodoView = Backbone.View.extend({
+    tagName: 'li',
+    className: 'todoContainer',
+    events: {
+        'keypress .todo': 'updateTodo',
+        'click .deleteTodo': 'deleteTodo',
+        'click .todoCheckBox': 'markComplete',
+    },
+
+    initialize: function(){
+        // this.todo = new TodoModel();
+        this.render();
+    },
+
+    render: function(){
+        var todo = this.model.toJSON();
+        this.el.innerHTML = todoViewTemplate({subject: todo.subject});
+    },
+
+    updateTodo: function(e){
+        var code = e.keyCode
+        var newSubject = e.currentTarget.value;
+
+        if (code === 13) {
```

Comment:
Same comment about either adding a global variable to this file or creating a constants.js file.